README.md

```diff
@@ -271,12 +271,11 @@ To display the footnote number instead of an ellipsis, provide the following `bu
 ```html
 <button
   aria-expanded="false"
-  aria-label="Footnote <% number %>"
   class="littlefoot-footnote__button"
   id="<% reference %>"
   title="See Footnote <% number %>"
 />
-  <% number %>
+  Footnote <% number %>
```

Comment:
I'd leave just the number. This makes the visible label for the footnote "Footnote X", which is rather noisy visually.

Reply:
Of course, no problem!